frontend/src/views/Files.vue

```diff
@@ -101,7 +117,9 @@ export default {
       this.$store.commit("closeHovers");
 
       // Set loading to true and reset the error.
-      this.setLoading(true);
+      if (window.sessionStorage.getItem('listFrozen') !=="true"){ 
+        this.setLoading(true);
+      }
```

Comment:
this.setLoading(true) is the function to reload the listing. we just disable it when the routing is not from a dir to a dir.